src/picongpu/CMakeLists.txt

```diff
@@ -164,29 +164,41 @@ endif(PARAM_OVERWRITES)
 # load cuda_memtest project
 ################################################################################
 
-set(SAME_NVCC_FLAGS_IN_SUBPROJECTS OFF)
-find_path(CUDA_MEMTEST_DIR
-        NAMES CMakeLists.txt
-        PATHS "${CMAKE_CURRENT_SOURCE_DIR}/../../thirdParty/cuda_memtest"
-        DOC "path to cuda_memtest"
-        )
+option(CUDAMEMTEST_ENABLE "Build cuda_memtest and the helper mpiInfo \
+                          (allow GPU health test before running PIConGPU)" ON)
+
+if("${PMACC_CUDA_COMPILER}" STREQUAL "clang")
```

Comment:
Can you please additionally describe the problems and what needs to be done via a new issue on https://github.com/ComputationalRadiationPhysics/cuda_memtest/issues ? Then link this issue please, too.